pyscf/md/integrators.py

```diff
@@ -202,11 +211,14 @@ def __init__(self, method, **kwargs):
         self.verbose = self.mol.verbose
         self.steps = 1
         self.dt = 10
+        self.T = None
```

Comment:
The temperature, tau, and temp_output don't need to go into the base `_Integrator` class but should only be placed into the subclass.

Reply:
Done!